packages/frontend/src/hooks/useSufficientBalance.ts

```diff
@@ -6,15 +6,15 @@ import { toTokenDisplay } from 'src/utils'
 export function useSufficientBalance(
   token?: Token,
   amount?: BigNumber,
-  estimatedGasCost?: BigNumber,
+  estimatedGasCost: BigNumber = BigNumber.from(500_000),
```

Comment:
Is the default estimatedGasCost `500_000` wei (`0.0000000000005` ETH) correct? seems kind of low

Reply:
thanks for the catch! i must have confused this with gas limit